app/components/react-grid.tsx

```diff
@@ -0,0 +1,309 @@
+    },
+    "& .react-grid-item > .react-resizable-handle::after": {
+      content: '""',
+      position: "absolute",
+      right: "3px",
+      bottom: "3px",
+      width: "5px",
+      height: "5px",
+      borderRight: "2px solid rgba(0, 0, 0, 0.4)",
+      borderBottom: "2px solid rgba(0, 0, 0, 0.4)",
+    },
+    "& .react-resizable-hide > .react-resizable-handle": {
+      display: "none",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-sw": {
+      bottom: "0",
+      left: "0",
+      cursor: "sw-resize",
+      transform: "rotate(90deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-se": {
+      bottom: "0",
+      right: "0",
+      cursor: "se-resize",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-nw": {
+      top: "0",
+      left: "0",
+      cursor: "nw-resize",
+      transform: "rotate(180deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-ne": {
+      top: "0",
+      right: "0",
+      cursor: "ne-resize",
+      transform: "rotate(270deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-w, & .react-grid-item > .react-resizable-handle.react-resizable-handle-e":
+      {
+        top: "50%",
+        marginTop: "-10px",
+        cursor: "ew-resize",
+      },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-w": {
+      left: "0",
+      transform: "rotate(135deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-e": {
+      right: "0",
+      transform: "rotate(315deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-n, & .react-grid-item > .react-resizable-handle.react-resizable-handle-s":
+      {
+        left: "50%",
+        marginLeft: "-10px",
+        cursor: "ns-resize",
+      },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-n": {
+      top: "0",
+      transform: "rotate(225deg)",
+    },
+    "& .react-grid-item > .react-resizable-handle.react-resizable-handle-s": {
+      bottom: "0",
+      transform: "rotate(45deg)",
+    },
+    "& .react-grid-item:not(.react-grid-placeholder)": {
+      background: theme.palette.background.paper,
+      border: theme.palette.divider,
+      boxShadow: theme.shadows[1],
+    },
+    "& .react-grid-item.static": {
+      background: "#cce",
+    },
+    "& .react-grid-item .text": {
+      fontSize: "24px",
+      textAlign: "center",
+      position: "absolute",
+      top: "0",
+      bottom: "0",
+      left: "0",
+      right: "0",
+      margin: "auto",
+      height: "24px",
+    },
+    "& .react-grid-item .minMax": {
+      fontSize: "12px",
+    },
+    "& .react-grid-item .add": {
+      cursor: "pointer",
+    },
+    "& .react-grid-dragHandleExample": {
+      cursor: "move",
+    },
+  },
+}));
+
+export const ChartGridLayout = (props: Props) => {
```

Comment:
Extra-nit: This is a personal preference, but I find it nice when props are defined just above the component, so we don't have to search for them 🕵️‍♂️